src/comments.js

```diff
@@ -376,11 +382,9 @@ function printTrailingComment(commentPath, print, options, parentNode) {
   const contents = printComment(commentPath);
   const isBlock = comment.type === "Block" || comment.type === "CommentBlock";
 
-  if (
-    util.hasNewline(options.originalText, locStart(comment), {
+  if (util.hasNewline(options.originalText, locStart(comment), {
       backwards: true
-    })
-  ) {
+    })) {
```

Comment:
Something must have regressed with last element expansion, there are many bugs like this because of it. I'l going to bisect and try to figure out what's going on.